assets/less/appeals/dec24.less

```
@@ -0,0 +1,397 @@
+@import '../updates-style.less';
+
+/*-------------------------
+* Media Queries
+*--------------------------*/
+@xxs: 25rem; // 400px
+@xs: 30rem; // 480px
+@ss: 33rem; // 528px ("sorta small")
+@sm: 40rem; // 640px
+@md: 48rem; // 768px
+@lg: 64rem; // 1024px
+@xl: 80rem; // 1280px
+@xxl: 90rem; // 1440px
+
+:root {
+
+  --bg-appeal-glow: linear-gradient(130deg, rgba(255, 242, 0, 0.25) 0%, #26BAF5 100%);
```

Glow doesn't really translate correctly from figma, so try this:
```suggestion
  --bg-appeal-glow: linear-gradient(130deg, rgba(255, 242, 0, 0.25) 50%, #26BAF5 100%);
```